lib/model/folderstate.go

```diff
@@ -38,7 +38,7 @@ func (s folderState) String() string {
 }
 
 type stateTracker struct {
-	folder string
+	folderId string
```

Comment:
It make sense when it is used as `f.folder`next to `f.dir`. Then the context not unveils its meaning. ;)